packages/shylu/shylu_node/tacho/src/impl/Tacho_NumericTools_LevelSet.hpp

```diff
@@ -1491,11 +1494,13 @@ class NumericToolsLevelSet : public NumericToolsBase<ValueType, DeviceType> {
             team_policy_update;
 #endif
 
-        functor_type functor(_info, _factorize_mode, _level_sids, _buf);
-
         team_policy_factor policy_factor(1, 1, 1);
         team_policy_update policy_update(1, 1, 1);
+        functor_type functor(_info, _factorize_mode, _level_sids, _buf);
 
+        // get max vector size
+        const ordinal_type factor_vmax = policy_factor.vector_length_max();
```

Comment:
You could also just have one "vmax" - it will always be the same for both policy tags